lib/View/login.dart

```diff
@@ -32,15 +35,10 @@ class _LoginScreenState extends State<LoginScreen> {
   void _updateButtonState() {
     final textLength = _phonenumbercontroller.text.length;
     setState(() {
-      _isButtonEnabled = textLength == 10;
+      _isButtonEnabled = textLength == 14;
     });
```

Comment:
nit: update the textLength to 14.
With this change the Submit button will only get enabled when the length of input text is 14